core-chart/templates/postcreatehooks/install-status-addon.yaml

```diff
@@ -0,0 +1,107 @@
+{{- if .Values.InstallPCHs }}
+apiVersion: tenancy.kflex.kubestellar.org/v1alpha1
+kind: PostCreateHook
+metadata:
+  name: install-status-addon
+  labels:
+    kflex.kubestellar.io/cptype: its
+spec:
+  templates:
+  - apiVersion: batch/v1
+    kind: Job
+    metadata:
+      name: "{{"{{.HookName}}"}}"
+    spec:
+      template:
+        spec:
+          initContainers:
+          - name: wait-for-its-hub-init
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
+            command:
+              - sh
+              - -c
+              - |
+                echo "Waiting for its-hub-init job to complete..."
+                kubectl wait --for=condition=complete --timeout=600s job/its-hub-init -n {{"{{.ControlPlaneName}}"}}-system
+                echo "its-hub-init job completed successfully"
+            env:
+            - name: KUBECONFIG
+              value: "{{"/etc/kube/{{.ITSkubeconfig}}"}}"
+            volumeMounts:
+            - name: kubeconfig
+              mountPath: "/etc/kube"
+              readOnly: true
+          - name: wait-for-ocm-crds
+            image: quay.io/kubestellar/kubectl:{{.Values.KUBECTL_VERSION}}
+            command:
+              - sh
+              - -c
+              - |
+                echo "Waiting for OCM CRDs to be established..."
+                # Wait for core OCM CRDs that clusteradm creates
+                kubectl wait --for=condition=Established crd/managedclusters.cluster.open-cluster-management.io --timeout=300s
+                kubectl wait --for=condition=Established crd/managedclustersets.cluster.open-cluster-management.io --timeout=300s
+                kubectl wait --for=condition=Established crd/managedclustersetbindings.cluster.open-cluster-management.io --timeout=300s
+                kubectl wait --for=condition=Established crd/placements.cluster.open-cluster-management.io --timeout=300s
+                kubectl wait --for=condition=Established crd/placementdecisions.cluster.open-cluster-management.io --timeout=300s
```

Comment:
Do we really need to wait for all of these?
Are all of these used while installing the status add-on?